pkg/commands/git_commands/commit.go

```diff
@@ -38,6 +38,24 @@ func (self *CommitCommands) SetAuthor(value string) error {
 	return self.cmd.New(cmdArgs).Run()
 }
 
+// Add a commit's coauthor using Github/Gitlab Co-authored-by metadata. Value is expected to be of the form 'Name <Email>'
+func (self *CommitCommands) AddCoAuthor(sha string, value string) error {
+	message, err := self.GetCommitMessage(sha)
+	if err != nil {
+		return err
+	}
+
+	message = message + fmt.Sprintf("\nCo-authored-by: %s", value)
```

Comment:
I think it's okay to leave this in English (as opposed to internationalising it). Do we know if github looks for this specific wording in order to render the commit differently?

Reply:
Yes, so in the commit message it looks for the metadata `Co-authored-by` and that's how it knows how to render the commit